Upstatement/blades
# added initials to missing images on about page

This PR inserts initials in the case of a missing icon:

![screen shot 2015-02-21 at 6 13 47 pm](https://cloud.githubusercontent.com/assets/1298086/6316459/86bd30aa-b9f5-11e4-9643-6977e8b6a650.png)

This way, when someone joins the company we don't have to wait for their logo to put them on the about page.

### To Test
- Create a new person with a missing icon and/or headshot:

![screen shot 2015-02-21 at 6 15 45 pm](https://cloud.githubusercontent.com/assets/1298086/6316461/ab215a02-b9f5-11e4-850f-6d105f340f73.png)

Observe that the initials will be substituted for the icon:

![screen shot 2015-02-21 at 6 16 43 pm](https://cloud.githubusercontent.com/assets/1298086/6316469/d533bda8-b9f5-11e4-9319-332064ee0e71.png)
